res/layout/upload_list_item.xml

```diff
@@ -31,12 +31,13 @@
         android:textColor="@color/black"
```

Comment:
this file was both used by Upload & Download fragment, so rename it to transfer_list_item, the same with fields inside.